<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
	<class>Report</class>
	<widget class="QScrollArea" name="Report">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>400</width>
				<height>300</height>
			</rect>
		</property>
		<property name="sizeIncrement">
			<size>
				<width>10</width>
				<height>10</height>
			</size>
		</property>
		<property name="windowTitle">
			<string>Student Performance Report</string>
		</property>
		<property name="verticalScrollBarPolicy">
			<enum>Qt::ScrollBarAsNeeded</enum>
		</property>
		<property name="horizontalScrollBarPolicy">
			<enum>Qt::ScrollBarAsNeeded</enum>
		</property>
		<property name="sizeAdjustPolicy">
			<enum>QAbstractScrollArea::AdjustToContents</enum>
		</property>
		<property name="widgetResizable">
			<bool>true</bool>
		</property>
		<widget class="QWidget" name="scrollAreaWidgetContents">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>398</width>
					<height>298</height>
				</rect>
			</property>
			<layout class="QVBoxLayout" name="verticalLayout_2">
				<item>
					<widget class="QLabel" name="StudentLabel">
						<property name="text">
							<string>{Student_name}: {DATE TIME}</string>
						</property>
						<property name="textFormat">
							<enum>Qt::PlainText</enum>
						</property>
						<property name="openExternalLinks">
							<bool>false</bool>
						</property>
						<property name="textInteractionFlags">
							<set>Qt::NoTextInteraction</set>
						</property>
					</widget>
				</item>
				<item>
					<layout class="QGridLayout" name="gridLayout">
						<item row="0" column="0">
							<widget class="QLabel" name="SuccLabel">
								<property name="text">
									<string>Passes:</string>
								</property>
							</widget>
						</item>
						<item row="0" column="1">
							<widget class="QLabel" name="ErrLabel">
								<property name="text">
									<string>Fails:</string>
								</property>
							</widget>
						</item>
						<item row="1" column="1">
							<widget class="QScrollArea" name="ErrorArea">
								<property name="verticalScrollBarPolicy">
									<enum>Qt::ScrollBarAsNeeded</enum>
								</property>
								<property name="sizeAdjustPolicy">
									<enum>QAbstractScrollArea::AdjustToContents</enum>
								</property>
								<property name="widgetResizable">
									<bool>true</bool>
								</property>
								<property name="alignment">
									<set>Qt::AlignCenter</set>
								</property>
								<widget class="QWidget" name="ErrList">
									<property name="enabled">
										<bool>true</bool>
									</property>
									<property name="geometry">
										<rect>
											<x>0</x>
											<y>0</y>
											<width>188</width>
											<height>254</height>
										</rect>
									</property>
									<property name="sizePolicy">
										<sizepolicy hsizetype="Expanding" vsizetype="Expanding">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<layout class="QVBoxLayout" name="verticalLayout_3">
										<item>
											<layout class="QVBoxLayout" name="ErrBox"/>
										</item>
									</layout>
								</widget>
							</widget>
						</item>
						<item row="1" column="0">
							<widget class="QScrollArea" name="PassArea">
								<property name="verticalScrollBarPolicy">
									<enum>Qt::ScrollBarAsNeeded</enum>
								</property>
								<property name="sizeAdjustPolicy">
									<enum>QAbstractScrollArea::AdjustToContents</enum>
								</property>
								<property name="widgetResizable">
									<bool>true</bool>
								</property>
								<property name="alignment">
									<set>Qt::AlignCenter</set>
								</property>
								<widget class="QWidget" name="PassList">
									<property name="geometry">
										<rect>
											<x>0</x>
											<y>0</y>
											<width>188</width>
											<height>254</height>
										</rect>
									</property>
									<property name="sizePolicy">
										<sizepolicy hsizetype="Expanding" vsizetype="Expanding">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<layout class="QVBoxLayout" name="verticalLayout">
										<item>
											<layout class="QVBoxLayout" name="PassBox"/>
										</item>
									</layout>
								</widget>
							</widget>
						</item>
					</layout>
				</item>
			</layout>
		</widget>
	</widget>
	<resources/>
	<connections/>
</ui>
